### PR TITLE
More shell fixes

### PR DESCRIPTION
**Pull request by aalexandrov**
*Description of changes:*

- Add proper documentation strings.
- Fix the printed script name in help output.
- Add a `force-static` train parameter that forces the creation of a static predictor from the expected model location.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by aalexandrov** [comment]
@jaheba we need to cut a new release after that. The change is not API breaking but adds a new parameter to a public API. Should we bump-up to 0.3.0 or 0.2.2?

**Comment by szha** [comment]
Job PR-185/1 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-185/1/index.html

**Comment by szha** [comment]
Job PR-185/2 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-185/2/index.html

**Comment by jaheba** [comment]
> @jaheba we need to cut a new release after that. The change is not API breaking but adds a new parameter to a public API. Should we bump-up to 0.3.0 or 0.2.2?

I think I prefer `0.2.2`.

File: src/gluonts/shell/__main__.py [hunk 1]
```diff
@@ -77,14 +77,39 @@ def cli() -> None:
...
-def serve_command(data_path: str, forecaster: Optional[str]) -> None:
+@click.option(
+    "--forecaster",
+    metavar='NAME',
+    envvar="GLUONTS_FORECASTER",
+    help=(
+        'An alias or a fully qualified name of a Predictor to use. '
+        'If this value is defined, the inference server runs in the '
+        'so-called dynamic mode, where the predictor is initialized for '
+        'each request using parameters provided by the in the "configuration" '
```
Comment (jaheba) [inline src/gluonts/shell/__main__.py]: "by the in the"

File: src/gluonts/shell/__main__.py [hunk 2]
```diff
@@ -77,14 +77,39 @@ def cli() -> None:
...
+        'mode, where the predictor is initialized upfront from a '
+        'serialized model located in the {data-path}/model folder.'
+    ),
+)
+@click.option(
+    "--force-static/--no-force-static",
+    envvar="GLUONTS_FORCE_STATIC",
+    default=False,
+    help=(
+        'Forces execution in static mode, even in situations where the '
```
Comment (jaheba) [inline src/gluonts/shell/__main__.py]: For a moment I thought static mode is when the `--forecaster` option is set. Maybe the names are a bit confusing and we should think about better ones in the future.

**Comment by szha** [comment]
Job PR-185/3 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-185/3/index.html

**Comment by szha** [comment]
Job PR-185/4 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-185/4/index.html

**Pull request merged**